### ISSUE TITLE
Update to headers-polyfill@3.1

### ISSUE THREAD

**Issue by aplr** [kettanaito/msw#1406]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Node.js version

v16

### Reproduction repository

-

### Reproduction steps

-

### Current behavior

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib' is not defined by "exports" in /myapp/node_modules/headers-polyfill/package.json
    at new NodeError (node:internal/errors:387:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:464:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:748:3)
    at resolveExports (node:internal/modules/cjs/loader:493:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:533:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:942:27)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/myapp/node_modules/msw/lib/node/index.js:658:18) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

Use `import ... from "headers-polyfill"` instead of `headers-polyfill/lib` in
- `[src/utils/request/MockedRequest.ts](https://github.com/mswjs/msw/blob/6c6e1193327ba6c7d55826fd7997ddc9f1fb4fd5/src/utils/request/MockedRequest.ts)`
- `[src/utils/request/MockedRequest.test.ts](https://github.com/mswjs/msw/blob/fc7f00c96e79ad0be83cbaf7c82ceff23f0c4355/src/utils/request/MockedRequest.test.ts)`
- `[src/handlers/GraphQLHandler.test.ts](https://github.com/mswjs/msw/blob/fc7f00c96e79ad0be83cbaf7c82ceff23f0c4355/src/handlers/GraphQLHandler.test.ts)`

### Expected behavior

No error

**Comment by kettanaito** [comment]
Hey, @aplr. Thanks for reporting this.

I'm already working on the fix (see #1407). Once the CI is green, will release today.

**Comment by aplr** [comment]
@kettanaito perfect, thanks. PR wasn't there when I opened the issue.

**Comment by kettanaito** [comment]
No worries.

**Comment by kettanaito** [comment]
## Released: v0.47.3 🎉

This has been released in v0.47.3!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v0.47.3)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/0.47.3)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
